### PR TITLE
[Ubuntu] Migrate tests to Pester

### PR DESCRIPTION
**Pull request by vsafonkin**
Added tests:
- mono, nuget
- mssql-cmd-tools
- pipx-packages
- r
- sbt
- selenium
- sphinx
- subversion
- swift
- swig
- terraform
- vcpkg

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by dibir-magomedsaygitov — APPROVED** [review]

File: images/linux/scripts/installers/r.sh [hunk 1]
```diff
@@ -12,3 +13,5 @@ sudo add-apt-repository "deb https://cloud.r-project.org/bin/linux/ubuntu $osLab
 sudo apt update
 sudo apt install r-base
 version=$(R --version | grep "R version" | cut -d " " -f 3)
```
Comment (al-cheb) [inline images/linux/scripts/installers/r.sh:15]: Should we delete this line?

**Review by al-cheb — APPROVED** [review]

File: images/linux/scripts/installers/r.sh (hunk 1 above)
Comment (vsafonkin) [reply images/linux/scripts/installers/r.sh]: I agree, deleted.

**Review by AlenaSviridenko — APPROVED** [review]

**Review by maxim-lobanov — APPROVED** [review]

**Review by dmitry-shibanov — APPROVED** [review]

File: images/linux/scripts/installers/mysql.sh [hunk 2]
```diff
@@ -4,6 +4,8 @@
 ##  Desc:  Installs MySQL Client
 ################################################################################
 
+source $HELPER_SCRIPTS/os.sh
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/mysql.sh:7]: How did it work without import before?

**Review by maxim-lobanov — APPROVED** [review]

File: images/linux/scripts/installers/mysql.sh (hunk 2 above)
Comment (vsafonkin) [reply images/linux/scripts/installers/mysql.sh:7]: @maxim-lobanov 
Screenshot 2020-12-28 at 11 34 34 AM

Comment (maxim-lobanov) [reply images/linux/scripts/installers/mysql.sh:7]: Nice

**Pull request merged**